plugin/action/throttle/throttle_test.go

```diff
@@ -14,6 +14,53 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type testConfig struct {
+	t           *testing.T
+	config      *Config
+	eventsTotal int
+	workTime    time.Duration
+}
+
+func (tconf *testConfig) runPipeline() {
```

Comment:
class variable should be named with one letter eg `func (c *testConfig) runPipeline() {`